libs/net/elastic/Models/Converters/AggregationsConverter.cs

```diff
@@ -105,7 +108,7 @@ private static IEnumerable<AggregationBucketModel> ParseAggregationBucketModels(
                 switch (propName.ToLower())
                 {
                     case var _ when propName.Equals(PropNameDocCount.ToLower()):
-                        bucket.DocCount = reader.GetInt64();
+                        bucket.DocCount = reader.GetInt64(); // unreliable
```

Comment:
If unreliable should you have it a TryParse instead?

Reply:
it's that the value is inherently wrong.  There might be a way to fix in the aggregation query itself, but this feels sane.
Part of the problem is that we have the "Not Applicable" Topic assigned to a valid "Topic Type". In the Elastic db that nested value is denormalized so when we group by TopicType and then TopicName, it doesnt care that "Issue:Not Applicable" has a child exclusion for "Not Applicable".  It just counts all the hits on "Issue".